usr/share/rear/layout/save/GNU/Linux/220_lvm_layout.sh

```diff
@@ -82,6 +82,9 @@ local lvs_exit_code
         # Skip lines that are not describing physical devices
         # i.e. lines where pdev does not start with a leading / character:
         test "${pdev#/}" = "$pdev" && continue
```

Comment:
@david-hill 
please add a comment that tells the reason behind
together with a URL to the matching issue like
```
        # Skip LVMs mounted on /dev/loop
        # see https://github.com/rear/rear/issues/2853
        if [[ "${pdev}" =~ '/dev/loop' ]]; then
            continue
        fi
```